uiowa/uiowa
# uiowa.edu - import redirects and match redirects

Resolves #1249

## To Test
- Pull uiowa.edu locally with a dsa
- Confirm that the redirect module is successfully patched
- Confirm that update hook importing redirects fires and completes.
- Login to the website to confirm the redirects table has been populated (several pages)
- Try a regular redirect and a match redirect (something/*)
